src/us/kbase/workspace/test/kbase/SampleServiceIntegrationTest.java

```diff
@@ -293,6 +317,18 @@ public void clearDB() throws Exception {
 		ARANGO.clearDatabase(ARANGO_DB, false);
 	}
 	
+
+	private SampleAddress createGenericSample() throws IOException, JsonClientException {
+		return SAMPLE_CLIENT.createSample(new CreateSampleParams()
```

Comment:
Jeez, I used to think only JS programmers used these crazy long chained functions...

Reply:
Fluent interfaces are all the rage in java land